packages/core/src/browser/core-commands.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License'); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import {
+    KeybindingContribution, KeybindingRegistry
+} from '../common/keybinding';
+import { KeyCode, Key, Modifier } from '../common/keys';
+import { CommandContribution, CommandRegistry } from '../common/command';
+import { Command } from '../common/command';
+import { injectable, inject } from 'inversify';
+import { FrontendApplication } from '../browser/frontend-application'
+
+export namespace CoreCommands {
+    export const TAB_NEXT: Command = {
+        id: 'tab:next',
+        label: 'Switch to next tab'
+    }
+    export const TAB_PREVIOUS: Command = {
+        id: 'tab:previous',
+        label: 'Switch to previous tab'
+    }
+}
+
+@injectable()
+export class CoreContribution implements KeybindingContribution, CommandContribution {
```

Comment:
There is already `CommonCommandContribution`, please use it instead:
- `commands-common.ts` should be moved to `browser` and renamed to `common-frontend-contribution`;
- `CommonCommandContribution` class should be renamed to `CommonFrontendContribution`, so it can implement `MenuContribution`, `CommandContribution` and `KeybindingContribution ` at once